### PR TITLE
Dynamically print future & past meetups based on date

### LINKED ISSUE #83: Automatically separate old vs future meetups based on date
As @nickorfas proposed we would use the date of each meetup to separate if it is a future meetup or an old one.

### PR DESCRIPTION
**Pull request by mzampetakis**
Using current date and meetups date, each meetup is presented as future or past meetup.

Fixes #83

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -88,13 +88,13 @@ <h2>The developers community<br>in Crete</h2>
 
   <!-- Upcomign Events -->
```
Comment (kouts) [inline index.html]: Can you fix this typo here?

File: index.html [hunk 2]
```diff
@@ -88,13 +88,13 @@ <h2>The developers community<br>in Crete</h2>
...
 	<div id="events">
-	  <div class="event" ng-if="MeetupList[0].IsFutureMeetup==1">
+	  <div class="event" ng-repeat="meetup in MeetupList" ng-if="isFutureMeetup(meetup.Day, meetup.Month, meetup.Year)">
```
Comment (kouts) [inline index.html]: This block needs 1 level deeper identation

File: index.html [hunk 3]
```diff
@@ -103,19 +103,19 @@ <h4>{{ MeetupList[0].Name }}</h4>
 		      <img src="images/calendar.png">
 		    </div>
 		    <div class="event_title date">
-		      <h4>{{ MeetupList[0].Day }}<span>/{{ MeetupList[0].Month }}</span></h4>
-		      <h3>{{ MeetupList[0].Year }}</h3>
+		      <h4>{{ meetup.Day }}<span>/{{ meetup.Month }}</span></h4>
+		      <h3>{{ meetup.Year }}</h3>
 		    </div>
 		  </div>
```
Comment (kouts) [inline index.html]: Consider removing this line.

Comment (kouts) [reply index.html]: Also valid for line 122

File: js/app.js [hunk 4]
```diff
@@ -60,6 +60,20 @@ controller('PastMeetupsController', function($scope) {
...
+        }
+        if (year==now.getFullYear() && month<now.getMonth()+1){
+            return false;
+        }
+        if (year==now.getFullYear() && month==now.getMonth()+1 && day<now.getDate()){
+            return false;
+        }
+        return true;
+    }
+
```
Comment (kouts) [inline js/app.js]: Consider removing this line

File: index.html (hunk 1 above)
Comment (mzampetakis) [reply index.html]: Fixed at the latest commit.

File: index.html (hunk 2 above)
Comment (mzampetakis) [reply index.html]: Fixed at the latest commit.

File: js/app.js (hunk 4 above)
Comment (mzampetakis) [reply js/app.js]: Fixed at the latest commit.

File: index.html (hunk 3 above)
Comment (mzampetakis) [reply index.html]: Fixed both at the latest commit.

**Comment by mzampetakis** [comment]
> I have noted some minor changes.
> Other than that I think that a `Date: "23-03-2020` (d-m-Y or d/m/Y) type of implementation would be more suitable for both readability purposes and lines of (data) code. But maybe that's just me, so just fixing these minor issues will suffice.
> 
> Great job!

I think that might be a future improvement. However, it might be harder to split date and then print it out.

**Pull request merged**